orbit/changes/bug-6935-cannot-run-orbit-shell-on-windows

```diff
@@ -0,0 +1 @@
+* Orbit does not exit now when running osquery shell
```

Comment:
does now exit?

Reply:
I've just fixed the change file as I made a mistake explaining the fixed bug